src/client/components/setting-panel/tree-list.jsx

```diff
@@ -23,7 +24,9 @@ import copy from 'json-deep-copy'
 import onDrop from './on-tree-drop'
 import Search from '../common/search'
 import Btns from './bookmark-transport'
+
 import './tree-list.styl'
+import time from '../../common/time';
```

Comment:
Make sure you run `npm i && npm run lint` before submit, you will get format tip from console if there are any lint errors.

Reply:
oh. It is auto input by ide. I will remove it.